themes/openy_themes/openy_rose/templates/node/node--blog--teaser.html.twig

```diff
@@ -81,26 +81,26 @@
 ] %}
 {% set category_default = node.bundle|clean_class|capitalize %}
 {% set category = content.field_blog_category.0 ? content.field_blog_category.0 : category_default|t %}
-{% set color = 'color: #' ~ content.field_blog_text_color['#items'].entity.field_color.value %}
-{% set bg_color = 'background-color: #' ~ content.field_blog_color['#items'].entity.field_color.value %}
+{% set color = content.field_blog_text_color['#items'].entity.field_color.value ? 'color: #' ~ content.field_blog_text_color['#items'].entity.field_color.value ~ ';' : null %}
```

Comment:
All changes in themes/openy_themes/openy_rose/templates/node/node--blog--teaser.html.twig not related to current issue, please remove them from PR